pr-data.csv

```diff
@@ -1,9 +1,9 @@
 Project URL,SHA Detected,Module Path,Fully-Qualified Test Name (packageName.ClassName.methodName),Category,Status,PR Link,Notes
-https://github.com/abel533/Mapper,1764748eedb2f320a0d1c43cb4f928c4ccb1f2f5,core,tk.mybatis.mapper.annotation.IdTest.testCompositeKeys,ID,Accepted,https://github.com/abel533/Mapper/pull/666,
-https://github.com/abel533/Mapper,1764748eedb2f320a0d1c43cb4f928c4ccb1f2f5,core,tk.mybatis.mapper.mapperhelper.ComplexEntityTest.test,ID,Accepted,https://github.com/abel533/Mapper/pull/666,
-https://github.com/abel533/Mapper,1764748eedb2f320a0d1c43cb4f928c4ccb1f2f5,core,tk.mybatis.mapper.mapperhelper.FieldHelperTest.testComplex,ID,Accepted,https://github.com/abel533/Mapper/pull/666,
-https://github.com/abel533/Mapper,1764748eedb2f320a0d1c43cb4f928c4ccb1f2f5,core,tk.mybatis.mapper.mapperhelper.FieldHelperTest.testUser,ID,Accepted,https://github.com/abel533/Mapper/pull/666,
-https://github.com/abel533/Mapper,1764748eedb2f320a0d1c43cb4f928c4ccb1f2f5,core,tk.mybatis.mapper.mapperhelper.SqlHelperTest.testLogicDeleteSql,ID,Accepted,https://github.com/abel533/Mapper/pull/666,
+https://github.com/abel533/Mapper,1764748eedb2f320a0d1c43cb4f928c4ccb1f2f5,core,tk.mybatis.mapper.annotation.IdTest.testCompositeKeys,ID,Opened,https://github.com/abel533/Mapper/pull/896,https://github.com/abel533/Mapper/pull/666
```

Comment:
Please add more info in `Notes`: https://github.com/abel533/Mapper/pull/666 was accepted but then reverted in commit  https://github.com/abel533/Mapper/commit/....

Reply:
I have added them Professor.